### PR TITLE
Flesh out Quick Start examples

### PR DESCRIPTION
**Pull request by mbidewell**
## Description
Update documentation to improve clarity of Blocking and Asynchronous Examples

## Type of change

Check all that apply

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (a change which changes the current internal or external interface)
- [X] This change requires a documentation update

## How Has This Been Tested?
This fix is for documentation changes
## Checklist:

- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation, if necessary
- [ ] My changes generate no new compiler warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Inspect the bytecode viewer, including reasoning why

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,19 +111,56 @@ Jitpack.io also allows to build from `fuel` forks. If a fork's username is `$you
...
+                    println(ex)
+                }
+                is Result.Success -> {
+                    val data = result.get()
+                    println(data)
+                }
+            }
+        }
+
+    httpAsync.join()
```
Comment (kittinunf) [inline README.md]: [nit] I don't think we have this function 🤣

**Comment by kittinunf** [comment]
Overall this is great! Thanks so much for updating the documentation for us!!!

File: README.md (hunk 1 above)
Comment (mbidewell) [reply README.md]: The join() call?  that comes from the CancellableRequest returned by responseString()

Comment (kittinunf) [reply README.md]: Oh yeah! I didn't know what I was thinking ... I thought this was the synchronous call!

**Pull request merged**